client/src/battle/battle_simulation.rs

```diff
@@ -9,6 +9,7 @@ use crate::resources::*;
 use crate::saves::Card;
 use framework::prelude::*;
 use generational_arena::Arena;
+use num_traits::ToPrimitive;
```

Comment:
Not sure if this import is necessary

Reply:
It was complaining about that but I don't remember adding it so I didn't touch it. Wanted to ask about it.